### PR TITLE
Feature/erp order package

### PR DESCRIPTION
**Pull request by julianzimmermann**

### CODE REVIEW

File: bundles/erp-order/src/FondOfOryx/Zed/ErpOrder/ErpOrderDependencyProvider.php [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        foreach ($plugins as $plugin) {
+            if (($plugin instanceof $class) === false) {
+                throw new WrongInterfaceException(sprintf(
+                    'Plugin %s has to implement interface from type %s',
+                    get_class($plugin),
+                    $class
+                ));
+            }
+        }
+    }
```
Comment (sourcecube) [inline bundles/erp-order/src/FondOfOryx/Zed/ErpOrder/ErpOrderDependencyProvider.php:233]: Sweet 👌  .. `InvalidInterfaceException` might be a good fit as well..

File: bundles/erp-order/src/FondOfOryx/Zed/ErpOrder/Persistence/Propel/Schema/erp_order.schema.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        <column name="id_erp_order_item" required="true" type="INTEGER" autoIncrement="true" primaryKey="true"/>
+
+        <column name="fk_erp_order" required="true" type="INTEGER"/>
+
+        <column name="sku" required="true" size="255" type="VARCHAR"/>
+        <column name="name" required="true" size="255" type="VARCHAR"/>
+        <column name="canceled_quantity" required="false" type="INTEGER"/>
+        <column name="ordered_quantity" required="false" type="INTEGER"/>
+        <column name="shipped_quantity" required="false" type="INTEGER"/>
+        <column name="invoice_quantity" required="false" type="INTEGER"/>
```
Comment (sourcecube) [inline bundles/erp-order/src/FondOfOryx/Zed/ErpOrder/Persistence/Propel/Schema/erp_order.schema.xml:57]: All other fields are past tense except `invoice_quantity` is that as wanted ?

File: dandelion.json [hunk 3]
```diff
@@ -13,13 +13,18 @@
         "erp-order": {
             "url": "https://<GITHUB_TOKEN>@github.com/fond-of-oryx/erp-order.git",
             "path": "bundles/erp-order/",
-            "version": "1.0.0"
+            "version": "alpha-0.0.1"
```
Comment (sourcecube) [inline dandelion.json:16]: I'm not sure but I believe the right way for semantic versioning would be as follow: 

`$major . $minor . $patch - $metalabel` 
Example: `1.0.0-alpha.0` or `1.0.0-alpha.1`

**Review by sourcecube — APPROVED** [review]

File: bundles/erp-order-extension/tests/_bootstrap.php [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+$pathToAutoloader = codecept_root_dir('vendor/autoload.php');
+
+if (!file_exists($pathToAutoloader)) {
+    $pathToAutoloader = codecept_root_dir('../../autoload.php');
+}
+
+require_once $pathToAutoloader;
```
Comment (daniel-rose) [inline bundles/erp-order-extension/tests/_bootstrap.php:9]:
```php
<?php

$pathToAutoloader = codecept_root_dir('vendor/autoload.php');

if (!file_exists($pathToAutoloader)) {
    $pathToAutoloader = codecept_root_dir('../../vendor/autoload.php');
}

if (!file_exists($pathToAutoloader)) {
    $pathToAutoloader = codecept_root_dir('../../autoload.php');
}

if (!file_exists($pathToAutoloader)) {
    $pathToAutoloader = codecept_root_dir('../../../../autoload.php');
}

require_once $pathToAutoloader;

```

File: bundles/erp-order/LICENSE [hunk 5]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 FOND OF GmbH
```
Comment (daniel-rose) [inline bundles/erp-order/LICENSE:3]: Maybe 2021?

**Review by daniel-rose — APPROVED** [review]

File: bundles/erp-order/src/FondOfOryx/Zed/ErpOrder/Persistence/Propel/Schema/erp_order.schema.xml (hunk 2 above)
Comment (daniel-rose) [reply bundles/erp-order/src/FondOfOryx/Zed/ErpOrder/Persistence/Propel/Schema/erp_order.schema.xml:57]: maybe invoiced_quantity?

**Review by gengjozsef — APPROVED** [review]

**Pull request merged**